test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
```

Comment:
Please avoid wildcard imports as per coding standards. You were doing so well! :(